snap/snapcraft.yaml

```diff
@@ -114,27 +124,29 @@ apps:
 package-repositories:
   - type: apt
     ppa: elmer-csc-ubuntu/elmer-csc-ppa
-  - type: apt
-    ppa: openscad/releases
```

Comment:
Why is openscad deleted ?

Reply:
The last release in that PPA is for focal. core22 is Jammy, openscad is in the official repositories for jammy so you will see further down I added it to build/stage packages in the freecad part.